src/app/user/group.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
```

Comment:
Please move this file to `src/app/group/group.service.ts` to comply with Angular conventions